pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,24 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        data_df = pd.DataFrame([{'col_a': '1', 'col_b': '16.5%',
+                                 'col_c': 'test'},
+                                {'col_a': '2.2', 'col_b': '15.3',
+                                 'col_c': 'another_test'}
+                                ])
+        reference_df = pd.DataFrame([{'col_a': '1', 'col_b': '16.5%',
+                                     'col_c': 'test'},
+                                     {'col_a': '2.2', 'col_b': '15.3',
+                                     'col_c': 'another_test'}
+                                     ], dtype='float64')
+        type_dict = {'col_a': 'float64', 'col_b': 'float64', 'col_c': 'object'}
+
+        with not_raises(ValueError):
+            df_astype = data_df.astype(dtype=type_dict, errors='ignore')
+
```

Comment:
you do't need the context manager at all, if this raises the test suite will simply fail

Reply:
I'm aware and that was how I wrote the original test case...there was a call to be more explicit in the tests and this was towards that effort..I'm left confused by these conflicting comments on this PR.

Reply:
Removed in 1c35cc70108a0407d964c26b5cfc07d654dd4ebc